clang/test/SemaCXX/derived-to-base-volatile-qual.cpp

```diff
@@ -0,0 +1,23 @@
+// RUN: %clang_cc1 -std=c++20 -fsyntax-only -ast-dump %s | FileCheck %s
```

Comment:
Should we also have a CodeGen test showing that we do a `store/load volatile`?

Reply:
Added a test in CodeGen too, thanks!